opencti-platform/opencti-graphql/tests/03-streams/00-Raw/raw-test.js

```diff
@@ -25,9 +25,9 @@ describe('Raw streams tests', () => {
       expect(createEventsByTypes.label.length).toBe(15);
       expect(createEventsByTypes.identity.length).toBe(31);
       expect(createEventsByTypes.location.length).toBe(15);
-      expect(createEventsByTypes.relationship.length).toBe(132);
+      expect(createEventsByTypes.relationship.length).toBe(136);
```

Comment:
Are you sure we need to change de raw numbers here ? Where this number is coming from ?
I thought we said to delete everything after your test will be run to avoid changing numbers. Is this because it's the manager and his behaviour is different ?

Reply:
after the test, all created elements are deleted, but creation events are taken into account